src/Bridge/Symfony/DependencyInjection/EonxUtilsExtension.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Utils\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+
+final class EonxUtilsExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
+    public function load(array $configs, ContainerBuilder $container): void
+    {
+        $fileLocator = new FileLocator(__DIR__ . '/../Resources/config');
+
+        (new XmlFileLoader($container, $fileLocator))->load('services.xml');
```

Comment:
I think we should try to avoid doing `new` and actions in the same line in case there is an error with one or the other, the logs will point to the line but it's unclear whether it's an issue with the instantiation or the command without investigation. We should probably think of this the same as AAA in testing.

Reply:
Eeeks!! Fixed :-)